src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -231,24 +212,22 @@ const TableHeader: React.FC<TableHeaderProps> = ({
         <div className="table-header__lock-indication">
           {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
         </div>
-        <label className="switch table-header__lock-toggle">
-          <input
-            type="checkbox"
-            className="switch-hidden-input"
-            checked={!!isCurrentTableLocked}
-            onChange={toggleIsCurrentTableLocked}
-          />
-          <span className="slider" />
-        </label>
+        <Toggler
+          toggled={!!isCurrentTableLocked}
+          containerClassName="table-header__lock-toggle"
+          onChange={toggleIsCurrentTableLocked}
+        />
       </div>
 
       <EditTableTitleModal
         isShown={isShown}
         title={tableTitle}
-        error={error}
+        description={tableSubtitle}
+        capacity={tableCapacity}
+        error={updateResponse.error?.message ?? ""}
```

Comment:
Instead of `?? ""` here, `error` should be an optional prop

Reply:
Makes sense.